### PR TITLE
Feature/asset view panel

### LINKED ISSUE #61: Asset browser panel

### PR DESCRIPTION
**Pull request by roeyb1**
Closes #61

### CODE REVIEW

File: EngineTests/Core/StringUtils.cpp [hunk 1]
```diff
@@ -32,4 +32,31 @@ TEST(StringUtils, BytesString)
 	TestSize = 1310720; // 2^20 + (2^20)/4
 	TestSizeString = StringUtils::CreateBytesString(TestSize);
 	EXPECT_STREQ(TestSizeString.c_str(), "1.25 MB"); // 1.13 = 1 + 1/4 (rounded to two decimal)
+}
+
+TEST(StringUtils, PathConcat)
```
Comment (filrod) [inline EngineTests/Core/StringUtils.cpp:37]: Are we keeping this test in our source code?

**Review by filrod — APPROVED** [review]
It works and it looks pretty slick too

File: EngineTests/Core/StringUtils.cpp (hunk 1 above)
Comment (roeyb1) [reply EngineTests/Core/StringUtils.cpp:37]: Yes of course, it's part of the `EngineTests` which is a set of unit tests which we run during CI.

Comment (filrod) [reply EngineTests/Core/StringUtils.cpp:37]: Oh sorry ya that makes a lot of sens :P

**Pull request merged**